feature1/file1.txt

```diff
@@ -0,0 +1 @@
+Is a new file
```

Comment:
Would be better if it said something else

Reply:
I did